go.mod

```diff
@@ -40,3 +40,17 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a // indirect
 )
+
+replace github.com/blevesearch/zapx/v15 => ../zapx
```

Comment:
Remove all these replaces.

Reply:
Yeah just got pushed to the review as part of "git add .". Removing them.